docs/general/getting-started.md

```diff
@@ -119,10 +119,10 @@ See the video below made by [DeFi Teller](https://defiteller.guru/) to know more
   </a>
   </td>
   <td>
-    <a class="guide-link" href="../docs/maintain-guides-how-to-join-council#submit-candidacy">  
```

Comment:
Instead of this, I think the "Make proposals" link should be modified to "Governance". It is odd to see "Make proposals" takes you to Treasury proposals and there is also a separate section on Treasury!

We need to find a good topic to replace "council candidacy" soon after OpenGov goes live.